opentech/apply/categories/templates/categories/admin/includes/meta_category_list_header.html

```diff
@@ -4,6 +4,10 @@
   <span>
     <span class="inline-block" style="margin-left:{{ depth }}em; font-size:{% if depth is 1 %}90{% elif depth is 2 %}80{% else %}100{% endif %}%;"></span>
     <i class="icon icon-fa-level-up icon-fa-rotate-90" style="display: inline-block;"></i>
-    {{ name }}
+    {% if is_archived %}
+        <span style="color: LightGray">{{ name }} (archived)</span>
```

Comment:
Use a class instead of hardcoding a style.

Reply:
This template is rendered to a [string](https://github.com/OpenTechFund/opentech.fund/blob/master/opentech/apply/categories/models.py#L66) and has no external stylesheet. And wagtail admin has its own styles. Adding inline styles is quick for overriding the listing header.